spec/factories/classifications.rb

```diff
@@ -21,13 +17,7 @@
     project
     workflow
     subject_ids do
-      if build_real_subjects
-        #TODO: clean this up -> this cascading create_list always builds:
-        # 7 Projects, 3 Workflows, 2 SubjectSets, 2 Subjects
-        create_list(:set_member_subject, 2).map(&:subject).map(&:id)
-      else
-        (1..10).to_a.sample(2)
-      end
+      create_list(:set_member_subject, 2).map(&:subject).map(&:id)
```

Comment:
will this run every time we build the factory.. it creates a very large number of models that aren't always needed, hence i added a transient feature.